numpy/core/src/multiarray/cblasfuncs.c

```diff
@@ -133,7 +133,8 @@ syrk(int typenum, enum CBLAS_ORDER order, enum CBLAS_TRANSPOSE trans,
 
             for (i = 0; i < n; i++) {
                 for (j = i + 1; j < n; j++) {
-                     *((npy_double*)PyArray_GETPTR2(R, j, i)) = *((npy_double*)PyArray_GETPTR2(R, i, j));
+                     *((npy_double*)PyArray_GETPTR2(R, j, i)) = \
```

Comment:
Just to clarify, macros need line continuations because the C preprocessor isn't very sophisticated. A continuation is also needed if you break inside a string, but that isn't needed as strings separated by whitespace -- line breaks are white space -- are combined, i.e., "Hello " "World" is "Hello World". Basically, the `\` just removes the line break from the input.